merge-request-status.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Creates a new column for merge request status.
+ *
+ * @type {{createElement: (function(): HTMLElement)}}
+ */
+import { utils } from "./common.js";
+
+const mergeRequestStatus = {
+  createElement: function () {
+    // store name elements in array-like object
+    const namesFromDOM = document.querySelectorAll("tbody .views-field-title a");
+    for (const nameElement of namesFromDOM) {
+      const issueId = utils.getIssueIdFromUrl(nameElement.getAttribute('href'));
+
+      const address = fetch(`https://git.drupalcode.org/api/v4/merge_requests?state=opened&scope=all&in=title&search=` + issueId)
```

Comment:
I am pretty new to this so I could be wrong but I **think** you can't actually make a request to another domain from a content script. in previous extension I made I had to do this from a background script.

Reply:
Also even in the background script you may have to add `https://git.drupalcode.org/*` to the `host_permissions` in the manifest.json file.

see the jira extension I link you for an example

Reply:
@narendra-drupal it maybe because the response from the server sets `Access-Control-Allow-Origin: *` . so the call to `https://git.drupalcode.org/api/v4` might not run into the problem I was expecting

Reply:
I also see now why I though there was problem with `data.length === 0`. Some of the response will not have any merge requests because there is not on the issue. So I now I see in the console the request that came back with matches